test-ai-review.sh

```
@@ -0,0 +1,163 @@
+MAX_RETRIES=2
+SUCCESS=false
+
+for MODEL in "${MODELS[@]}"; do
+  echo ""
+  echo ">>> Trying: $MODEL"
+
+  for ATTEMPT in $(seq 1 $MAX_RETRIES); do
+    echo "    Attempt $ATTEMPT/$MAX_RETRIES"
+
+    TMP=$(mktemp)
+    if timeout 120 $TOOL_BIN run -m "$MODEL" < "$PROMPT_FILE" > "$TMP" 2>"$OPENCODE_ERR"; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)